src/App.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import './App.scss';
+import TodoList from './components/TodoList/TodoList';
+import AddTodoForm from './components/AddTodoForm/AddTodoForm';
+
+import users from './api/users';
+import todos from './api/todos';
+
+const getUserById = userId => users.find(user => user.id === userId);
+
+const preparedTodos = [...todos].map(todo => ({
```

Comment:
```suggestion
const preparedTodos = todos.map(todo => ({
```
It's not necessary to create a copy of the initial array. `map` method isn't mutating original array